packages/gestalt/src/Image.js

```diff
@@ -22,126 +23,107 @@ type Props = {|
   srcSet?: string,
 |};
 
-export default class Image extends PureComponent<Props> {
-  static propTypes = {
-    alt: PropTypes.string.isRequired,
-    children: PropTypes.node,
-    color: PropTypes.string,
-    fit: PropTypes.oneOf(['contain', 'cover', 'none']),
-    importance: PropTypes.oneOf(['high', 'low', 'auto']),
-    loading: PropTypes.oneOf(['eager', 'lazy', 'auto']),
-    naturalHeight: PropTypes.number.isRequired,
-    naturalWidth: PropTypes.number.isRequired,
-    onError: PropTypes.func,
-    onLoad: PropTypes.func,
-    sizes: PropTypes.string,
-    src: PropTypes.string.isRequired,
-    srcSet: PropTypes.string,
-  };
-
-  static defaultProps: {|
-    color: string,
-    fit?: 'contain' | 'cover' | 'none',
-    importance?: 'high' | 'low' | 'auto',
-    loading?: 'eager' | 'lazy' | 'auto',
-  |} = {
-    color: 'transparent',
-    fit: 'none',
-    importance: 'auto',
-    loading: 'auto',
-  };
-
-  componentDidMount() {
```

Comment:
is this one inside 'useEffect'? `useEffect` has a different conditinal `shouldScaleImage(this.props.fit))`

Reply:
@AlbertCarreras line 70 combines the previous `componentDidMount` and `componentDidUpdate`.